sphinx/api/configuration/rose-configuration-format.rst

```diff
@@ -13,9 +13,9 @@ following:
 * (optionally) files containing data that cannot easily be represented by the
   INI format.
 
-We have added the following conventions into the rose configuraiton format:
+We have added the following conventions into the Rose configuraiton format:
```

Comment:
I've noticed a typo ('configura**i**ton') - it's outside of the PR changes but we should get it fixed now.